### PR TITLE
SEX TOYS

### PR DESCRIPTION
**Pull request by Memeji**
# Description

This PR adds several vibrators, fleshlights, and dildos to the station. These can even be bought in the cargo catalogue for 1500 spessos!

---

# Changelog

:cl:
- add: added Dildos
- add: added Fleshlights
- add: added Vibrators

### CODE REVIEW

**Comment by ShatteredSwords** [comment]
You made the linter unhappy with "ammount" instead of "amount"

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Tests fails are valid.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Test fails are still valid.

**Comment by Memeji** [comment]
> Test fails are still valid.

Yes, I am still in the process of finding where they hard-coded the limit.
If you have any ideas, lemme know.

File: Resources/Prototypes/Catalog/Fills/Lockers/misc.yml [hunk 1]
```diff
@@ -169,6 +171,42 @@
...
+        - id: HorseDildo
+          prob: 0.02
+        - id: DragonDildo
+          prob: 0.02
+        - id: NormalDildo
+          prob: 0.02
+        - id: TentacleDildo
+          prob: 0.02
+        - id: DoubleDildo
+          prob: 0.02   # Floof - End
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Catalog/Fills/Lockers/misc.yml:209]:
```suggestion
        - id: LewdWand # Floof - Start
          prob: 0.02
          orGroup: Lewd
        - id: LewdVibeGreen
          prob: 0.02
          orGroup: Lewd
        - id: LewdVibeTeal
          prob: 0.02
        - id: LewdVibePink
          prob: 0.02
        - id: LewdVibeRed
          prob: 0.02
          orGroup: Lewd
        - id: LewdVibeYellow
          prob: 0.02
        - id: LewdFleshlightGreen
          prob: 0.02
        - id: LewdFleshlightTeal
          prob: 0.02
          orGroup: Lewd
        - id: LewdFleshlightPink
          prob: 0.02
          orGroup: Lewd
        - id: LewdFleshlightRed
          prob: 0.02
          orGroup: Lewd
        - id: LewdFleshlightYellow
          prob: 0.02
          orGroup: Lewd
        - id: AvianDildo
          prob: 0.02
          orGroup: Lewd
        - id: CanineDildo
          prob: 0.02
          orGroup: Lewd
        - id: HorseDildo
          prob: 0.02
          orGroup: Lewd
        - id: DragonDildo
          prob: 0.02
          orGroup: Lewd
        - id: NormalDildo
          prob: 0.02
          orGroup: Lewd
        - id: TentacleDildo
          prob: 0.02
          orGroup: Lewd
        - id: DoubleDildo
          prob: 0.02   # Floof - End
          orGroup: Lewd
```

File: Resources/Prototypes/Catalog/Fills/Lockers/misc.yml [hunk 2]
```diff
@@ -264,6 +302,42 @@
...
+        - id: HorseDildo
+          prob: 0.02
+        - id: DragonDildo
+          prob: 0.02
+        - id: NormalDildo
+          prob: 0.02
+        - id: TentacleDildo
+          prob: 0.02
+        - id: DoubleDildo
+          prob: 0.02  # Floof - End
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Catalog/Fills/Lockers/misc.yml:340]:
```suggestion
        - id: LewdWand  # Floof - Start
          prob: 0.02
          orGroup: Lewd
        - id: LewdVibeGreen
          prob: 0.02
          orGroup: Lewd
        - id: LewdVibeTeal
          prob: 0.02
          orGroup: Lewd
        - id: LewdVibePink
          prob: 0.02
          orGroup: Lewd
        - id: LewdVibeRed
          prob: 0.02
          orGroup: Lewd
        - id: LewdVibeYellow
          prob: 0.02
          orGroup: Lewd
        - id: LewdFleshlightGreen
          prob: 0.02
          orGroup: Lewd
        - id: LewdFleshlightTeal
          prob: 0.02
          orGroup: Lewd
        - id: LewdFleshlightPink
          prob: 0.02
          orGroup: Lewd
        - id: LewdFleshlightRed
          prob: 0.02
          orGroup: Lewd
        - id: LewdFleshlightYellow
          prob: 0.02
          orGroup: Lewd
        - id: AvianDildo
          prob: 0.02
          orGroup: Lewd
        - id: CanineDildo
          prob: 0.02
          orGroup: Lewd
        - id: HorseDildo
          prob: 0.02
          orGroup: Lewd
        - id: DragonDildo
          prob: 0.02
          orGroup: Lewd
        - id: NormalDildo
          prob: 0.02
          orGroup: Lewd
        - id: TentacleDildo
          prob: 0.02
          orGroup: Lewd
        - id: DoubleDildo
          prob: 0.02  # Floof - End
          orGroup: Lewd
```

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Test fails are still valid.

1 too much item.

**Review by FoxxoTrystan — APPROVED** [review]

**Pull request merged**